lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -131,24 +62,28 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_DAYS, M
    * @function
    * @public
    *
+   * @param {String} version - version
+   * @param {Object} options - options
+   * @param {Boolean} [options.allowSleepUpdateCheck=true] - allow sleeping the update check
```

Comment:
Functions using Angular modals are incredibly hard to test, plus this option is only used on the angular template (which is also very hard to test), so I guess there is not much we can do with this for now. Can we address it after the react migration?